src/ngraph_encapsulate_op.cc

```diff
@@ -66,13 +67,22 @@ class NGraphEncapsulateOp : public tf::OpKernel {
     // TODO(amprocte): need to check status result here.
     OP_REQUIRES_OK(ctx, tf::ConvertGraphDefToGraph(opts, *graph_def, &m_graph));
 
+     auto env_value = std::string(std::getenv("NGRAPH_TF_BACKEND")); 
     // Create the backend
     if (m_ng_backend == nullptr) {
-#if defined(NGRAPH_EMBEDDED_IN_TENSORFLOW)
-      m_ng_backend = std::make_shared<ng::runtime::interpreter::INTBackend>();
-#else
-      m_ng_backend = ng::runtime::Backend::create("CPU");
-#endif
+	if( env_value  == "INTERPRETER") {
+      		m_ng_backend = ng::runtime::Backend::create("INTERPRETER");
+	}
+	else if( env_value == "CPU") {
+      		m_ng_backend = ng::runtime::Backend::create("CPU");
+	}
+	else if( env_value == "NNP") {
+      		m_ng_backend = ng::runtime::Backend::create("NNP");
+	}
+	else if( env_value == "GPU") {
```

Comment:
Right now the GPU won't work unless some changes are made in the data handling part. So please remove this option until that time.